src/test/java/apoc/export/cypher/ExportCypherEnterpriseFeaturesTest.java

```diff
@@ -0,0 +1,136 @@
+        driver = GraphDatabase.driver(neo4jContainer.getBoltUrl(), AuthTokens.none());
+        session = driver.session();
+        session.writeTransaction(tx -> {
+            tx.run("CREATE CONSTRAINT ON (t:Person) ASSERT (t.name, t.surname) IS NODE KEY;");
+            tx.success();
+            return null;
+        });
+        session.writeTransaction(tx -> {
+            tx.run("CREATE (a:Person {name: 'John', surname: 'Snow'}) " +
+                    "CREATE (b:Person {name: 'Matt', surname: 'Jackson'}) " +
+                    "CREATE (c:Person {name: 'Jenny', surname: 'White'}) " +
+                    "CREATE (d:Person {name: 'Susan', surname: 'Brown'}) " +
+                    "CREATE (e:Person {name: 'Tom', surname: 'Taylor'})" +
+                    "CREATE (a)-[:KNOWS]->(b);");
+            tx.success();
+            return null;
+        });
+    }
+
+    @AfterClass
+    public static void afterAll() {
+        if (neo4jContainer != null) {
+            session.close();
+            driver.close();
+            neo4jContainer.close();
+        }
+        cleanBuild();
+    }
+
+    @Test
+    public void testExportWithCompoundConstraintCypherShell() throws Exception {
+        String fileName = "testCypherShellWithCompoundConstraint.cypher";
+        File output = getFile(fileName);
+        testCall(session, "CALL apoc.export.cypher.all(null,{config})",
+                map("file", getFilePath(output), "config", Util.map("format", "cypher-shell"), "stream", isTravis()), (r) -> {
+                    assertExportStatement(EXPECTED_CYPHER_SHELL_WITH_COMPOUND_CONSTRAINT, output, r);
+                });
+    }
+
+    @Test
+    public void testExportWithCompoundConstraintPlain() throws Exception {
+        File output = getFile("testPlainFormatWithCompoundConstraint.cypher");
+        testCall(session, "CALL apoc.export.cypher.all({file},{config});",
+                map("file", getFilePath(output), "config", Util.map("format", "plain"), "stream", isTravis()),
+                (r) -> assertExportStatement(EXPECTED_PLAIN_FORMAT_WITH_COMPOUND_CONSTRAINT, output, r));
+    }
+
+    @Test
+    public void testExportWithCompoundConstraintNeo4jShell() throws Exception {
+        File output = getFile("testNeo4jShellWithCompoundConstraint.cypher");
+        testCall(session, "CALL apoc.export.cypher.all({file},{config});",
+                map("file", getFilePath(output), "config", Util.map("format", "neo4j-shell"), "stream", isTravis()),
+                (r) -> assertExportStatement(EXPECTED_NEO4J_SHELL_WITH_COMPOUND_CONSTRAINT, output, r));
+    }
+
+    private void assertExportStatement(String expectedStatement, File output, Map<String, Object> result) {
+        assertEquals(expectedStatement, isTravis() ? result.get("cypherStatements") : readFile(output));
+    }
+
+    private static String readFile(File output) {
+        try {
+            return new Scanner(output).useDelimiter("\\Z").next() + String.format("%n");
```

Comment:
we should move those into utilities or use it from there.
also in Java8 we should probably rather use something from java.nio.Files https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html#lines-java.nio.file.Path-